src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -647,40 +647,39 @@ public void SetMembrane(string membraneName)
         gui.UpdateMembraneButtons(Membrane.InternalName);
     }
 
-    public void SetRigidity(float rigidity)
+    public void SetRigidity(int rigidity)
     {
-        if (Math.Abs(Rigidity - rigidity) < MathUtils.EPSILON)
+        int intRigidity = (int)Math.Round(Rigidity * 10);
```

Comment:
I'd really like you replace all places with `10` in the changes with a named constant in Constants.